notebooks/sparse_embedding_retrieval.ipynb

```diff
@@ -0,0 +1,10657 @@
+{
```

Comment:
<p>"Here we are initializing a *<strong>FastembedSparseDocumentEmbedder</strong> to generate...<strong>" + </strong>in the title</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-cookbook/pull/79/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
I didn't quite understand the comment, but I tried to express the concept better and without repetitions.

Reply:
<p>I meant to tell you to remove the space between "FastEmbed Sparse Document Embedder" and make it <strong>FastembedSparseDocumentEmbedder </strong>instead. You need to apply the change to the section title as well hence " + in the title". Is there a reason why you want to have a space between words? I'd suggest to make the first letters lower case if you want to keep it that way</p><p> </p><p>Thanks for providing more info 🙌</p>

Reply:
ah!

done.